src/utils/Processing.js

```diff
@@ -22,6 +63,9 @@ export function checkForDatabaseUpgrade(dependencyArr, databaseName = 'FSH Playg
       database = event.target.result;
       let existingObjectStores = database.objectStoreNames;
       helperReturn.version = database.version;
+      if (existingObjectStores.contains('resources')) {
+        helperReturn.shouldUpdate = true;
```

Comment:
The answer to this might be yes, but is there a need to keep checking the other if/else case if we hit this case? Or can we make this part of the same if case as the other two and have this close the database and resolve if we reach this first case?